src/Manager.hpp

```diff
@@ -49,6 +51,7 @@ class ECS {
 		this->entityIndex = 0;
 	}
 
+    // CreateEntity will initialize and return a new entity with no components.
```

Comment:
Unsure if its github review or not but this comment doesnt look tabbed in enough

Reply:
Fixed, along with another couple lines